test/test_puma_server.rb

```diff
@@ -952,4 +947,11 @@ def test_request_body_wait_chunked
 
     assert request_body_wait >= 1000
   end
+
+  def test_siginfo_prints_thread_traces
```

Comment:
need to fix this test name as this particular test has nothing to do with the signal